### PR TITLE
Send banquet email on invite

### PR DESCRIPTION
**Pull request by e-martinsson**
## Describe your changes

Fixes: #1066 

Also adds a button to manually send an invitation mail.

## Checklist before requesting review

- [x] Feature/fix PRs should add one atomic (as small as possible) feature or fix.
- [x] The code compiles and all the tests pass.

### CODE REVIEW

File: banquet/views.py [hunk 1]
```diff
@@ -1058,6 +1101,64 @@ def invitation(request, year, token):
...
+    return render(
+        request,
+        "banquet/invite_sent.html",
+        {
+            "fair": fair,
+        },
+    )
+
+
+def send_invitation_mail(name, date, location, link, email):
```
Comment (didrikmunther) [inline banquet/views.py:1142]: Does this also contain a link to the invitation, so that you can press the link and easily get the QR code?

Comment (e-martinsson) [reply banquet/views.py:1142]: image
Yes!

File: templates/banquet/manage_invitation.html [hunk 2]
```diff
@@ -39,6 +39,7 @@ <h2>Invitation link</h2>
 	{% else %}
 		<p>This invitation is tied to a user account. Ask {{ invitation.user.get_full_name }} to sign in to the AIS and click Banquet in the menu.</p>
 	{% endif %}
+	<a href="{% url 'banquet_send_invitation_button' fair.year banquet.pk invitation.pk %}" class="btn btn-default">Send invitation mail</a>
```
[the remark this answers is not in the document]
Comment (e-martinsson) [reply templates/banquet/manage_invitation.html]: Fixed

**Review by didrikmunther — APPROVED** [review]
Fantastic!

**Pull request merged**